SPECS/clamav/clamav.spec

```diff
@@ -12,9 +12,9 @@ Source0:        https://github.com/Cisco-Talos/clamav/archive/refs/tags/%{name}-
 # To update the cache run:
```

Comment:
Need to revisit this comment. How were the vendor sources re-created for this update?